config/initializers/git_revision.rb

```diff
@@ -1,8 +1,6 @@
 module AppInfo
   if Rails.env.production?
-    #TODO: git SHA available once devops PR #271 is merged:
-    #https://github.com/department-of-veterans-affairs/devops/pull/271
-    GIT_REVISION = "unknown"
+    GIT_REVISION = ENV["APP_GIT_SHA"]
```

Comment:
There's a separate issue open for this already: https://github.com/department-of-veterans-affairs/platform-team/issues/19

Your change would break in production (where git is not installed) if we forget to set the `APP_GIT_SHA` ENV variable.